ckanext/canada/templates/internal/header.html

```diff
@@ -1,64 +1,89 @@
-                  'inventory',
-                  'dac',
-                  'nap',
-                  'experiment',
-                  ] -%}
-                  {# XXX bit of a hack: we "know" that our dataset types have
-                     resource names with the same values #}
-                  <li><a href="{{ h.url_for('recombinant_type', resource_name=r_type)
-                    }}">{{ _(h.recombinant_get_geno(r_type).title) }}</a></li>
-                {%- endfor -%}
-              </ul>
-            </li>
-            {% endif %}
-            <li><a href="{{ h.url_for(controller='package', action='search') }}" class="item">{{ _('Search') }}</a></li>
-            <li><a href="{{ h.url_for(controller='ckanext.canada.controller:CanadaController', action='organization_index') }}" class="item">{{ _('Organizations') }}</a></li>
-            <li><a href="{{ h.url_for(controller='ckanext.canada.controller:CanadaController', action='view_help') }}" class="item">{{ _('FAQ') }}</a></li>
-            {% if c.userobj %}
-            <li><a href="{{ h.url_for(controller='user', action='logout') }}" class="item">{{ _('Log out') }}</a></li>
-            {% else %}
-            <li><a href="{% url_for controller='user', action='login' %}" class="item">{{ _('Log in') }}</a></li>
-            {% endif %}
-        {% endblock %}
-      </ul>
+    <nav role="navigation" id="wb-sm" class="visible-md visible-lg" typeof="SiteNavigationElement">
+        <div class="container navbar">
+            <h2>Topics menu</h2>
+            <div class="row">
+                <ul class="list-inline menu">
+                    {% block header_site_navigation_tabs %}
+                <li role="presentation"><a role="menuitem"  href="{{ _('http://open.canada.ca') }}">{{ _('Open.Canada.ca') }}</a></li>
+                {% if h.check_access('package_create') %}
+                <li class="dropdown">
+                    <a data-toggle="dropdown" class="dropdown-toggle" href="#">{{ _('Add to Portal') }} <b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li class="dropdown-item"><a role="menuitem" tabindex="-1" href="{{ h.url_for('dataset_new') }}">{{ _('Open Data') }}</a></li>
+                        <li class="dropdown-item"><a role="menuitem" tabindex="-1" href="{{ h.url_for('info_new') }}">{{ _('Open Information') }}</a></li>
+                        {%- for r_type in [
+                          'ati',
+                          'contracts',
+                          'contractsa',
+                          'hospitalityq',
+                          'travelq',
+                          'travela',
+                          'grants',
+                          'reclassification',
+                          'wrongdoing',
+                          'briefingt',
+                          'consultations',
+                          'inventory',
+                          'dac',
+                          'nap',
```

Comment:
did you intend to remove the experiment type as part of this PR?

Reply:
Oh no. I don't know when that disappeared. Thanks for the catch!